src/Nox.Types.Extensions/Types/LanguageCode/LanguageExtensions.cs

```diff
@@ -0,0 +1,25 @@
+using Nox.Reference;
+
+namespace Nox.Types.Extensions;
+
+/// <summary>
+/// Class containing extension methods for the <see cref="Language"/> class.
+/// </summary>
+public static class LanguageExtensions
+{
+    /// <summary>
+    /// Retrieves the reference language code for the specified <paramref name="language"/>.
+    /// </summary>
+    /// <param name="language">The <see cref="Language"/> object.</param>
+    /// <returns>The reference language code as a <see cref="LanguageCode"/> object.</returns>
+    /// <exception cref="ArgumentNullException">Thrown when <paramref name="language"/> or <paramref name="language"/>'s Iso_639_1 value is null.</exception>
+    public static LanguageCode GetReferenceLanguageCode(this Language language)
+    {
+        if (language.Iso_639_1 is not null)
```

Comment:
I would suggest to decrease complexity vu having something like:
```suggestion
        ArgumentNullException.ThrowIfNull(language, nameof(language));
        ArgumentNullException.ThrowIfNull(language.Iso_639_1, nameof(language.Iso_639_1));
        return LanguageCode.From(language.Iso_639_1);
```